docs/reference/ingest/processors/data-stream-router.asciidoc

```diff
@@ -0,0 +1,43 @@
+[[data-stream-router-processor]]
+=== Data stream router processor
+++++
+<titleabbrev>Data stream router</titleabbrev>
+++++
+
+The `data_stream_router` processor allows to route a document from one data stream to another data stream.
+It can use both static values or values from the document to determine the target data stream.
+
+The name of a data stream is comprised of three parts and looks like this: `<type>-<dataset>-<namespace>`.
+See the {fleet-guide}/data-streams.html#data-streams-naming-scheme[data stream naming scheme] documentation for more details.
+
+NOTE: `data_stream_router` processor can only be used on data streams that follow the data streams naming scheme.
```

Comment:
We should add the link to https://www.elastic.co/blog/an-introduction-to-the-elastic-data-stream-naming-scheme here

Reply:
No. There are some mentions of it in the docs but it also links to the blog post.

Reply:
Seems like we do have docs an I've linked it on line 11: https://www.elastic.co/guide/en/fleet/current/data-streams.html#data-streams-naming-scheme